doc/api/repl.markdown

```diff
@@ -106,6 +108,14 @@ the following values:
 You can use your own `eval` function if it has following signature:
 
     function eval(cmd, context, filename, callback) {
+      var result;
+      try {
+        result = vm.runInThisContext(cmd);
+      } catch (e) {
+        if (isRecoverableError(e)) {
```

Comment:
Already updated below. I'm only using `error.name` because it's assumed the `try..catch` would have an error object already to check.